components/RadiolabFooter.vue

```diff
@@ -66,41 +66,52 @@ onMounted(() => {
 .site-footer {
   section {
     background-color: var(--blue-500);
+    padding-bottom: var(--player-height);
```

Comment:
add space for the audio player , so it does not cover footer content. We can make this dynamic in the future based on if anything is playing or not